packages/expo-background-fetch/ios/EXBackgroundFetch/EXBackgroundFetchTaskConsumer.m

```diff
@@ -20,6 +20,12 @@ - (NSString *)taskType
 - (void)didRegisterTask:(id<EXTaskInterface>)task
 {
   _task = task;
+  [self updateMinimumInterval];
+}
+
+- (void)setOptions:(NSDictionary *)options
+{
+  [self updateMinimumInterval];
```

Comment:
Just by looking at this diff, shouldn't we do something with these options?

Reply:
No, we can access to them by `_task.options` if needed